lilac2/lilacpy.py

```diff
@@ -80,6 +81,10 @@ def load_lilac(dir: Path) -> Generator[LilacMod, None, None]:
       for i, entry in enumerate(mod.update_on):
         if 'alias' in entry:
           mod.update_on[i] = ALIASES[entry['alias']]
+        elif entry.get('source') == 'alpm-lilac':
+          entry['source'] = 'alpm'
+          entry['dbpath'] = PACMAN_DB_DIR
+          entry.setdefault('repo', _G.repo.name)
```

Comment:
Making it in `aliases.yaml` will have better visitibility, and we fill in `dbpath` here for that alias.

Reply:
Makes sense. However, this alias is a bit special because we should allow users to fill the repo and package name. Probably we can make a general improvement to allow adding & overriding arguments for aliases? I think it will make the aliases more flexible and is unlikely to break existing things.

Reply:
Yes, we can merge additional keys with alias content. But either the path needs to be hard-coded, or we do variable expansions on alias values.

If you do it, don't forget to update the `pre-commit` check script.

Reply:
Just implemented variable expansion for alias values.